packages/oxlint-config/src/index.test.ts

```
@@ -310,9 +316,15 @@ describe("oxlint-config", () => {
           "unicode-bom": ["error", "never"],
           "use-isnan": "error",
           "valid-typeof": "error",
-        },
+        }),
       });
 
+      const reactDoctorRules = Object.entries(frontend.rules ?? {}).filter(([ruleName]) =>
```

ESLint

[error] Avoid having conditionals in tests

(jest/no-conditional-in-test)